src/components/SearchIcon/SearchIcon.component.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+
+const pathStyles = {
```

Comment:
This styles can be turned into styled components to keep concistency

Reply:
Done, refactored the svg to become a styled component